src/Service.Host/client/src/actions/serialNumberTransactionUtilityActions.js

```diff
@@ -0,0 +1,31 @@
+import { CALL_API } from 'redux-api-middleware';
+import config from '../config';
+import * as sharedActionTypes from './index';
+
+const getSernosTransCodes = () => ({
+    [CALL_API]: {
```

Comment:
Could this not use one of the standard actions?
If it needs to stand alone then CALL_API should be replaced by RSAA (CALL_API is deprecated).

Reply:
will merge master in and resolve